cmd/termui/provider/logProvider.go

```diff
@@ -0,0 +1,91 @@
+package provider
+
+import (
+	"encoding/hex"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/logger"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/gorilla/websocket"
+)
+
+var formatter = logger.PlainFormatter{}
+var webSocket *websocket.Conn
+
+// InitLogHandler will open the websocket and set the log level
+func InitLogHandler(nodeURL string, logLevel string) error {
+	var err error
+	webSocket, err = openWebSocket(nodeURL, fmt.Sprintf("*:%s", logLevel))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func openWebSocket(address string, logLevelPatterns string) (*websocket.Conn, error) {
+	u := url.URL{
+		Scheme: "ws",
```

Comment:
might use the same mechanism as in cmd/logviewer/main.go L2019-212, also might add the required flag

Reply:
+1

Reply:
done